src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -918,12 +918,13 @@ private void ReferToTempIfReferenceTypeReceiver(BoundLocal receiverTemp, ref Bou
         }
 
         /// <summary>
-        /// Patch refKinds for arguments that match 'In' or 'Ref' parameters to have effective RefKind.
+        /// Patch refKinds for arguments that match 'in', 'ref', or 'ref readonly' parameters to have effective RefKind.
         /// For the purpose of further analysis we will mark the arguments as -
-        /// - In        if was originally passed as None
-        /// - StrictIn  if was originally passed as In
-        /// - Ref       if the argument is an interpolated string literal subject to an interpolated string handler conversion. No other types
-        ///             are patched here.
+        /// - In                    if was originally passed as None or Ref
```

Comment:
>In                    if was originally passed as None or Ref [](http://example.com/codeflow?start=14&length=61)

This doesn't look accurate. It looks like, if an argument is passed as `ref`, then its effective RefKind is 'ref', `ref readonly`, or `strict in`. The comment looks confusing in general because it doesn't include how parameter RefKind is involved into the process, but it makes a difference.